app.json

```diff
@@ -8,13 +8,21 @@
         "required": true
     },
     "GUILD_ID": {
-        "description": "The id for the server you are hosting this bot for.",
+        "description": "The id for the server you are serving this bot for.",
         "required": true
     },
+    "MODMAIL_GUILD_ID": {
+        "description": "Operations center for the server you are serving for.",
```

Comment:
Describe that this config var is only necessary if you want the bot to have a separate operations center.

Reply:
Fixing

Reply:
Try: `The server which is the operations center for the bot. This is only necessary if you want a separate server in which modmail operations occur.`